Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -5071,6 +5080,10 @@ - (void)reloadViewItems
             }
         }
 
+        if (viewItem.interaction.timestampForSorting > collapseCutoffTimestamp) {
+            shouldHideFooter = NO;
```

Comment:
Only collapse footers for any new messages (message created since we enter the conversation) once per minute.